integration-tests/main/pom.xml

```diff
@@ -18,6 +18,8 @@
         <datasource.url>jdbc:h2:tcp://localhost/mem:test</datasource.url>
         <datasource.username></datasource.username>
         <datasource.password></datasource.password>
+
+        <webjar.jquery-ui.version>1.13.0</webjar.jquery-ui.version>
```

Comment:
As long as dependabot doesn't pick this up I'm fine with this. Could add a little note though.

Reply:
I improved things. Let's merge.